components/devtools_traits/lib.rs

```diff
@@ -359,5 +360,5 @@ impl PreciseTime {
     }
 }
 
-#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, MallocSizeOf, PartialEq, Serialize)]
-pub struct WorkerId(pub u32);
+#[derive(Clone, Debug, Deserialize, Eq, Hash, MallocSizeOf, PartialEq, Serialize)]
+pub struct WorkerId(pub String);
```

Comment:
You can keep uuid and implement malloc_size_of, or maybe use u128 (https://docs.rs/uuid/0.8.1/uuid/struct.Uuid.html#method.to_u128_le).